packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1175,7 +1176,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           onMouseLeave={this._onOptionMouseLeave}
           role="option"
           aria-selected={isSelected ? 'true' : 'false'}
-          ariaLabel={this._getPreviewText(item)}
+          ariaLabel={ariaLabel}
```

Comment:
The `getPreviewText` added for some specific cases and the but this PR is trying to address shouldn't be fixed like this.